README.md

```diff
@@ -126,7 +126,9 @@ and you'll be given the list of sample options.
 #### Production APIs
 
 - [**Production Build**](./production): Build code in advance for faster Worker startup times.
-- [**Debugging**](https://docs.temporal.io/application-development/debugging#debug): The [replay-history](./replay-history) sample shows how to retrieve Workflow Event History and debug it using the `runReplayHistory` Worker API ([video](https://youtu.be/fN5bIL7wc5M)).
+- [**Debugging**](https://docs.temporal.io/application-development/debugging#debug):
+  - The [replay-history](./replay-history) sample shows how to retrieve Workflow Event History and debug it using the `runReplayHistory` Worker API ([video](https://youtu.be/fN5bIL7wc5M)).
```

Comment:
I would just remove it at this point and (if there's a reference to it from YT) note in the video description that the sample has been replaced by this newer and shinier one.